selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/WorkspacesListTest.java

```diff
@@ -209,89 +176,37 @@ public void checkWorkspaceSelectingByCheckbox() throws Exception {
   }
 
   @Test(groups = UNDER_REPAIR)
-  public void checkSorting() {
-    workspaces.waitPageLoading();
-    workspaces.clickOnRamButton();
-
-    List<Workspaces.WorkspaceListItem> items = workspaces.getVisibleWorkspaces();
-
-    // items are sorted by name, check is present for ensuring of items order
-    if (items.get(0).getRamAmount() != BLANK_WS_MB) {
-      workspaces.clickOnRamButton();
-      items = workspaces.getVisibleWorkspaces();
-    }
+  public void checkSearchField() throws Exception {
+    int nameLength = WORKSPACE_NAME.length();
+    int existingWorkspacesCount = getWorkspacesCount();
+    String sequenceForSearch = WORKSPACE_NAME.substring(nameLength - 5, nameLength);
 
-    // check items order after "RAM" clicking
-    try {
-      assertEquals(items.get(0).getRamAmount(), BLANK_WS_MB);
-      assertEquals(items.get(1).getRamAmount(), JAVA_WS_MB);
-    } catch (AssertionError ex) {
-      // remove try-catch block after issue has been resolved
-      fail("Known permanent failure https://github.com/eclipse/che/issues/4242");
-    }
+    workspaces.waitVisibleWorkspacesCount(existingWorkspacesCount);
+    workspaces.typeToSearchInput(sequenceForSearch);
 
-    // check reverse order after "RAM" clicking
-    workspaces.clickOnRamButton();
-    items = workspaces.getVisibleWorkspaces();
     try {
-      assertEquals(items.get(0).getRamAmount(), JAVA_WS_MB);
-      assertEquals(items.get(1).getRamAmount(), BLANK_WS_MB);
-    } catch (AssertionError ex) {
+      workspaces.waitVisibleWorkspacesCount(EXPECTED_SORTED_WORKSPACES_COUNT);
```

Comment:
Question about name `EXPECTED_SORTED_WORKSPACES_COUNT` - I can't find statement to sort projects. 
Does it mean "EXPECTED_FOUND_WORKSPACES_COUNT`?